entries/SWC-106.md

```diff
@@ -1,16 +1,16 @@
 # Title 
-Unprotected SUICIDE Instruction
+Unprotected SELFDESTRUCT/SUICIDE Instruction
 
 ## Base/Class Type
 [CWE-284: Improper Access Control](https://cwe.mitre.org/data/definitions/284.html)
 
 ## Description 
 
-Due to missing or insufficient access controls, malicious parties can kill the contract account.
+Due to missing or insufficient access controls, malicious parties can self destruct the contract.
 
 ## Remediation
 
+Self destructing a contract has to be considered carefully and it should be verified if it is required. In cases where there is valid use case it is recommened to use a MultiSig wallet and ensure that the action is authorized by multiple parties.
```

Comment:
Consider removing the self-destruct functionality unless it is absolutely required. If there is valid use-case, it is recommended to implement multisig functionality so that multiple parties must approve the self-destruct action.